backend/tests/test_document.py

```diff
@@ -205,9 +96,9 @@ def test_update_status(client):
     db.session.commit()
 
     rs = client.put(
-        "/document/status",
-        content_type="application/json",
-        json={"status": "Missing", "docID": temp_document.id},
+        "/document/status/" + str(temp_document.id),
+        content_type="multipart/form-data",
+        data={"status": "Missing"},
```

Comment:
As a result of using FormData, our test syntax needs to change. Instead of `content_type="application/json`, we need to use `content_type="multipart/form-data"`, and instead of `json={...}`, we need to use `data={...}`.